Source/Scene/Cesium3DTileset.js

```diff
@@ -90,7 +90,7 @@ define([
      * @constructor
```

Comment:
There are a bunch of examples in the documentation that need to be fixed. Search for `url : 'http://localhost:8002/tilesets/Seattle'` - I couldn't find other cases.